.github/CHANGELOG.md

```diff
@@ -468,6 +468,9 @@ random_mat1 = rng.random((3,2))
 random_mat2 = rng.standard_normal(3, requires_grad=False)
 ```
 
+* Ising YY gate functionality added.
+  [(#1358)](https://github.com/PennyLaneAI/pennylane/pull/1358)
+
```

Comment:
Hi @paniash, could you move this addition to the top of the changelog where we have the details of version 0.17.0-dev? At the beginning of the week, we have entered a feature freeze stage for the PennyLane v0.16.0 release, which means that this new addition will come up in v0.17.0. :slightly_smiling_face: Also, don't forget to add yourself as one of the first contributors to the new release! :)

Once done, this PR is good to be merged.

Reply:
Awesome! Thanks. :-)

Reply:
Thanks! Could you also add yourself to the new list of contributors? It will be starting on line 19. Preferably one new line before and after the names will be great.

Reply:
I had made a commit to the repo and added myself as a contributor at the time. Do I need to do it again specifically for 0.17.0-dev?

Reply:
Yes, please. This contribution will be coming up in release 0.17.0. We have a group of contributors for each release separately, so in this case, you have contributed to both 0.16.0 and 0.17.0. We just need to mark the latter case too. :)

Reply:
Got it. Thanks!